tests/basic/customer_issues_test.go

```diff
@@ -757,3 +757,185 @@ var _ = Describe("{CreateCloudSnapAndDelete}", func() {
+	appVol, err := Inst().V.InspectVolume(volName.ID)
+	if err != nil {
+		return "", nil, err
+	}
+	attachedNode := appVol.AttachedOn
+	log.InfoD("Volume [%v] is attached to Node [%v]", volName.Name, attachedNode)
+
+	nodeDetails, err := node.GetNodeByIP(attachedNode)
+	if err != nil {
+		return "", nil, err
+	}
+
+	killPxExecErr := KillPxExecUsingPid(nodeDetails)
+	if killPxExecErr != nil {
+		return "", nil, err
+	}
+
+	killPxStorageErr := KillPxStorageUsingPid(nodeDetails)
+	if killPxStorageErr != nil {
+		return "", nil, err
+	}
+
+	return attachedNode, &nodeDetails, nil
+}
+
+func isPodStuckNotRunning(nameSpace string) (bool, map[string]string, error) {
+	restartDetails := make(map[string]string)
+	isPodRestarting := false
+	pods, err := GetAllPodsInNameSpace(nameSpace)
+	if err != nil {
+		log.InfoD("Get all pods from Namespace returned error")
+		return false, nil, err
+	}
+	for _, eachPod := range pods {
+		if eachPod.Status.Phase != "Running" && eachPod.Status.Phase != "ContainerCreating" && eachPod.Status.Phase != "Terminating" {
+			restartDetails[eachPod.Name] = fmt.Sprintf("%v", eachPod.Status.Phase)
+			log.InfoD("Pod Status of Pod [%v] is [%v]", eachPod.Name, eachPod.Status.Phase)
+		}
+	}
+	if len(restartDetails) > 0 {
+		isPodRestarting = true
+	}
+	return isPodRestarting, restartDetails, nil
+}
+
+var _ = Describe("{ContainerCreateDeviceRemoval}", func() {
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ContainerCreateDeviceRemoval",
+			"Test app stuck in Container Creation with Device exists",
+			nil, 0)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "App Stuck in ContainerCreation State with Device Exists in the backend"
+	It(stepLog, func() {
+
+		var isPodRestarting bool = false
+		var terminateScript bool = false
+
+		Inst().AppList = []string{"vdbench-sv4-svc"}
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("volwithdeviceexist-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		//defer appsValidateAndDestroy(contexts)
+
+		allVolumes, err := GetAllVolumesWithIO(contexts)
+		log.FailOnError(err, "Failed to get volumes with IO Running")
+
+		log.InfoD("List of volumes with IO Running [%v]", allVolumes)
+
+		// Get Random Volume with IO running on the cluster
+		var randomVol *volume.Volume
+		for _, eachVol := range allVolumes {
+			log.InfoD("Trying volume [%s]", eachVol)
+			if strings.Contains(eachVol.Name, "vdbench-pvc-enc-sv4-svc") {
+				randomVol = eachVol
+				break
+			}
+		}
+		log.Infof("Volume picked for testing device exist issue [%v]", randomVol)
+
+		appVol, err := Inst().V.InspectVolume(randomVol.ID)
+		allNodesForVolume := appVol.ReplicaSets[0].Nodes
+		log.InfoD("List of nodes for the volume [%s] are [%v]", randomVol.Name, allNodesForVolume)
+
+		go func(nameSpace string) {
+			defer GinkgoRecover()
+			for {
```

Comment:
seems there is no break in this loop. please have a look

Reply:
done